docker-compose.yml

```diff
@@ -829,3 +830,15 @@ services:
       timeout: 10s
       retries: 10
     logging: *logging
+
+  postgresql:
```

Comment:
Please move this entire section to the "Dependent Services" section of this file. It should be in that section alphabetically based on the service name, so around line 681.